auth-connector/src/generated/main/grpc/com/cloudera/thunderhead/service/usermanagement/UserManagementGrpc.java

```diff
@@ -2695,6 +2695,191 @@ private UserManagementGrpc() {}
      }
      return getSetWorkloadSubdomainMethod;
   }
+  @io.grpc.ExperimentalApi("https://github.com/grpc/grpc-java/issues/1901")
```

Comment:
just curious, why check in the generated file?

Reply:
@giladwolff not sure. I'm just following the existing pattern there.

Reply:
We usually publish the generated classes for the proto files as they don't really change. It's one less step to do gradle build to have that file.